app/views/observations/identify.html.haml

```diff
@@ -3,6 +3,7 @@
   = I18n.t :identify_title
 - content_for :extrajs do
   :javascript
+    var DISPLAYABLE_APP_IDS = #{ [OauthApplication.inaturalist_android_app&.id, OauthApplication.inaturalist_iphone_app&.id, OauthApplication.seek_app&.id] };
```

Comment:
This can potentially make a JS array that looks like `[nil,nil,nil]` which is not valid JS. I think you should probably stick `.compact.to_json` onto the end.